personio-personnel-data-api-oa3.yaml

```diff
@@ -997,11 +997,13 @@ paths:
                           id: 1234
                           name: 'Paid vacation'
                           category: null
+                          unit: 'day'
                       - type: TimeOffType
                         attributes:
                           id: 1235
                           name: 'Home office'
-                          category: offsite_work
+                          category: 'offsite_work'
```

Comment:
Will this may not break some implementations?

Reply:
This is just a minor change of type of the category. Meaning this is a string. 
before that, it wasn't set strictly, though, by default, it was still the string.